### PR TITLE
Remove MCSB Overrides in Assignment

### PR DESCRIPTION
**Pull request by Springstone**
## Overview/Summary

Cleanup of the Microsoft Cloud Security Benchmark initiative assignment parameters to align with all RIs. These parameters were set to disable a handful of policies in the initiative inconsistently across RIs and have now reverted to defaults.

Related to : https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/issues/528

## This PR fixes/adds/changes/removes

1. Removed parameters in the DINE-ASBPolicyAssignment.json to revert to initiative defaults.

### Breaking Changes

1. None

## Testing Evidence

Please provide any testing evidence to show that your Pull Request works/fixes as described and planned (include screenshots, if appropriate).

### Testing URLs

The below URLs can be updated where the placeholders are, look for `{YOUR GITHUB BRANCH NAME HERE - Remove Curly Brackets Also}` & `{YOUR GITHUB BRANCH NAME HERE - Remove Curly Brackets Also}`, to allow you to test your portal deployment experience.

> Please also replace the curly brackets on the placeholders `{}`

#### Azure Public

[Deploy To Azure](https://portal.azure.com/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2F{YOUR GITHUB ORG/ACCOUNT HERE - Remove Curly Brackets Also}%2FEnterprise-Scale%2F{YOUR GITHUB BRANCH NAME HERE - Remove Curly Brackets Also}%2FeslzArm%2FeslzArm.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2F{YOUR GITHUB ORG/ACCOUNT HERE - Remove Curly Brackets Also}%2FEnterprise-Scale%2F{YOUR GITHUB BRANCH NAME HERE - Remove Curly Brackets Also}%2FeslzArm%2Feslz-portal.json)

#### Azure US Gov (Fairfax)
[Deploy To Azure](https://portal.azure.us/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2F{YOUR GITHUB ORG/ACCOUNT HERE - Remove Curly Brackets Also}%2FEnterprise-Scale%2F{YOUR GITHUB BRANCH NAME HERE - Remove Curly Brackets Also}%2FeslzArm%2FeslzArm.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2F{YOUR GITHUB ORG/ACCOUNT HERE - Remove Curly Brackets Also}%2FEnterprise-Scale%2F{YOUR GITHUB BRANCH NAME HERE - Remove Curly Brackets Also}%2FeslzArm%2Ffairfaxeslz-portal.json)

## As part of this Pull Request I have

- [X] Checked for duplicate [Pull Requests](https://github.com/Azure/Enterprise-Scale/pulls)
- [ ] Associated it with relevant [issues](https://github.com/Azure/Enterprise-Scale/issues), for tracking and closure.
- [X] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Enterprise-Scale/tree/main)
- [ ] Performed testing and provided evidence.
- [ ] Updated relevant and associated documentation.
- [ ] Updated the ["What's New?"](https://github.com/Azure/Enterprise-Scale/wiki/Whats-new) wiki page (located: `/docs/wiki/whats-new.md`)

### CODE REVIEW

**Review by jtracey93 — CHANGES_REQUESTED** [review]
Hey @sacha

**Comment by Springstone** [comment]
@jtracey93 updated What's New section as part of this commit.

File: docs/wiki/Whats-new.md [hunk 1]
```diff
@@ -2,23 +2,91 @@
 
 - [In this Section](#in-this-section)
 - [Updates](#updates)
+  - [December 2022](#december-2022)
+    - [Policy](#policy)
   - [November 2022](#november-2022)
+    - [Docs](#docs)
+    - [Tooling](#tooling)
+  - [Policy](#policy-1)
+  - [Other](#other)
```
Comment (jtracey93) [inline docs/wiki/Whats-new.md:11]: @Springstone can you roll back the changes to the TOC in this page, apart from the new month of December being added?

This is normally a VSCode extension that does this like MarkdownLint etc. (disabling it normally fixes it)

**Review by jtracey93 — CHANGES_REQUESTED** [review]

File: docs/wiki/Whats-new.md (hunk 1 above)
Comment (Springstone) [reply docs/wiki/Whats-new.md]: Done @jtracey93 - had some PEBKAC this morning... sorry for the commits.

**Review by jtracey93 — APPROVED** [review]
LGTM 

cc: @krowlandson

**Pull request merged**